app/models/manageiq/providers/configuration_manager.rb

```diff
@@ -1,11 +1,7 @@
 class ManageIQ::Providers::ConfigurationManager < ::ExtManagementSystem
-  require_nested :InventoryGroup
-  require_nested :InventoryRootGroup
-
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
-  has_many :inventory_groups,             :dependent => :destroy, :foreign_key => "ems_id", :inverse_of => :manager
```

Comment:
@bdunne Should this stay in ConfigurationManager?  That is, would any config manager need it, and if not, why was it here in the base to begin with?